chapters/VK_QNX_external_memory_screen_buffer/qnx_screen_buffer.adoc

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 QNX Software Systems
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[[memory-external-qnx-screen-buffer]]
+==== QNX Screen Buffer
+
+The QNX SDP defines code:_screen_buffer objects, which represent
+a buffer that the QNX Screen graphics subsystem can use directly
+in its windowing system APIs. More specifically, a Screen buffer is an area of memory that
+stores pixel data. It can be attached to Screen windows, streams, or pixmaps.
+These QNX Screen buffer objects may: be imported into
+slink:VkDeviceMemory objects for access via Vulkan.
+An slink:VkImage or slink:VkBuffer can: be bound to the imported
+slink:VkDeviceMemory object if it is created with
+ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_SCREEN_BUFFER_BIT_QNX.
+
+--
+code:struct code:_screen_buffer is strongly typed, so naming the handle type is
+redundant. The internal layout and therefore size of a code:struct code:_screen_buffer
+image may depend on native usage flags that do not have corresponding Vulkan
+counterparts.
+--
+
+A slink:VkDeviceMemory imported from a QNX Screen buffer has no way to acquire
+a reference to its code:_screen_buffer object. Therefore, during the host
+execution of a Vulkan command that has a QNX Screen buffer as a parameter
+(including indirect parameters via pname:pNext chains), the application must:
+ensure that the QNX Screen buffer resource remains valid.
+
+[[memory-external-screen-buffer-external-formats]]
+===== QNX Screen Buffer External Formats
+
+QNX Screen buffers may: represent images using implementation-specific
+formats, layouts, color models, etc., which do not have Vulkan equivalents.
+Such _external formats_ are commonly used by external image sources such as
+video decoders or cameras.
+Vulkan can: import QNX Screen buffers that have external formats, but
+since the image contents are in an undiscoverable and possibly proprietary
+representation, images with external formats must: only be used as sampled
+images, must: only be sampled with a sampler that has {YCbCr} conversion
+enabled, and must: have optimal tiling.
```

Comment:
dose it means some vk APIs will fail if the back storage memory come from screen_buffer with external format? 
 for example, vkCreateFrameBuffer will fail ?